File: backend/src/main/java/com/coing/CoingApplication.java
package com.coing;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

import io.swagger.v3.oas.annotations.enums.SecuritySchemeType;
import io.swagger.v3.oas.annotations.security.SecurityScheme;

@SecurityScheme(
	name = "bearerAuth",
	type = SecuritySchemeType.HTTP,
	scheme = "bearer",
	bearerFormat = "JWT"
)

@SpringBootApplication
@EnableScheduling
@EnableJpaAuditing
public class CoingApplication {

	public static void main(String[] args) {
		SpringApplication.run(CoingApplication.class, args);
	}

}

File: backend/src/main/java/com/coing/CoingApplication.kt
package com.coing

import io.swagger.v3.oas.annotations.enums.SecuritySchemeType
import io.swagger.v3.oas.annotations.security.SecurityScheme
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.retry.annotation.EnableRetry
import org.springframework.scheduling.annotation.EnableScheduling

@SecurityScheme(name = "bearerAuth", type = SecuritySchemeType.HTTP, scheme = "bearer", bearerFormat = "JWT")
@SpringBootApplication
@EnableScheduling
@EnableJpaAuditing
@EnableRetry

class CoingApplication

fun main(args: Array<String>) {
    SpringApplication.run(CoingApplication::class.java, *args)
}

